### DISCUSSION TITLE
Notification badge

### DISCUSSION THREAD

**Discussion by robertjmccarthy** [ministryofjustice/moj-frontend#706] [RELEASED: Components, pages and patterns]
The [notification badge component](https://design-patterns.service.justice.gov.uk/components/notification-badge/) is live in the MoJ Design System.

Use this discussion to suggest changes and improvements to this component.

**Comment by davehaigh** [comment]
Small suggestion, the padding looks a little top heavy on this

It is currently 5px top and 2px bottom

3px top (see second pic) makes it look a little more vertically centred 

<img width="219" height="241" alt="image" src="https://github.com/user-attachments/assets/47de58fb-ef31-45a5-af07-9033875bfcec" />

<img width="225" height="233" alt="image" src="https://github.com/user-attachments/assets/1bcd23a5-8162-45c9-bb85-38d42cdd8bb8" />

**Comment by davehaigh** [comment]
Worth considering alternate colour usage overrides for these too (similar to the alert component has)

For example a blue variant, to draw attention to an item (for example a tab) without making it look like an error

<img width="311" height="174" alt="image" src="https://github.com/user-attachments/assets/2a540053-aaad-4bc9-a4f3-13a7146e8fdf" />

**Comment by davehaigh** [comment]
Another suggestion to consider if it has any value

Alternative content to numbers, for example the icons used on the alert component. This could also visually relate an alert to the sections they are trying to draw attention to.

<img width="460" height="153" alt="image" src="https://github.com/user-attachments/assets/75023a0f-91c4-4164-a47d-fa03f6892b1f" />

<img width="1212" height="438" alt="image" src="https://github.com/user-attachments/assets/ad9c800e-a625-4a56-a2d2-f2b70f79d7c8" />

**Comment by 1Paulmensah** [comment]
Here’s a new, improved version of the MoJ Design System notification badge.

### Breaking changes 💥
The notification badge now contains spacing on the left for adding it to a link. This may affect the layout of your service. You need to remove any spacing you’ve added to (or around) the component. This needs to be done in Figma designs and in code.
This makes sure it’s used consistently across MoJ.

### What’s new :nerd_face:
The documentation now includes instructions on:

- how to use it
- getting accessibility right

There are lots of examples of how it could look in your service.
We’d love 🥰 you to take a look - view the [notification badge on the MoJ Design System documentation website](https://design-patterns.service.justice.gov.uk/components/notification-badge).
